Update product-management-team-handbook.md (#722)
mattermost/mattermost-handbook: operations/research-and-development/product/pm-onboarding.md

A good place to start is exploring the websites, and then see if you can find a way to install a version of Mattermost to try it out.

* Google Mattermost
* [https://mattermost.com](https://mattermost.com/)
* [https://docs.mattermost.com](https://docs.mattermost.com/) \(User, Cloud, Self-Managed, Integrations guides\)
* [https://developers.mattermost.com](https://developers.mattermost.com/)
* [http://github.com/mattermost](https://github.com/mattermost/)
* [https://www.mattermost.com/demo](https://www.mattermost.com/demo/)

### Activities:

* Meet with direct manager
  * Company org overview
    * [https://mattermost.bamboohr.com/employees/orgchart.php](https://mattermost.bamboohr.com/employees/orgchart.php)
    * [https://developers.mattermost.com/internal/rd-teams/](https://developers.mattermost.com/internal/rd-teams/)
  * Add to meetings
  * Get accounts set up
  * First day projects
  * Set up 1-1 times and create 1-1 doc
* Meet with “PM buddy”
  * Learn about team
  * Add to team meetings
  * Set up 1-1 times and create 1-1 doc
* Look at the product with “fresh eyes”

>   * Install Mattermost on your laptop, go through the motions of setting it up

Reviewer: ```suggestion
  * Install Mattermost on your laptop, go through the steps of setting it up
```

    * Optionally also install the Mattermost Mobile App on your mobile device
  * As you read the documentation, look out for any improvements that can be made \(outdated information, spelling/grammar, etc\)

## First week

### Objective: Introduction to customers team members, resources, and make first contribution

### Training focus:

* Introduction to our Customer Obsession Principle
* Introductions to team members through 1-1s
  * Use introductions as learning opportunities to understand different parts of the company \(ask questions!\)
* Introduction to product
* Introduction to our “Fast, Obvious, Forgiving” design principles
* Introduction to key resources
* Make your first contribution on GitHub

### Activities:

Customer Obsession:
* Join and introduce yourself in these channels, ask to join any and all customer calls - see if you can get a couple scheduled this week:
  * [Product Management](https://community.mattermost.com/private-core/channels/product-management)

>   * [GTM:PM Engagement](https://community.mattermost.com/private-core/channels/csmpm-engagement) \(Stu, Jenn\)

Reviewer 2: ```suggestion
  * [GTM:PM Engagement](https://community.mattermost.com/private-core/channels/csmpm-engagement) \(Stu Doherty, Jenn Lawler\)
```

>   * [Sales Discussion](https://community.mattermost.com/private-core/channels/sales-discussion) \(Lance, Steve, Richard Pidgeon)

Reviewer 2: ```suggestion
  * [Sales Discussion](https://community.mattermost.com/private-core/channels/sales-discussion) \(Lance Howden, Steve Green, Richard Pidgeon)
```

> * Join and review the topics and posts in these channels to get a better understanding of our customers

Reviewer: ```suggestion
* Join and review the topics and posts in these channels to get a better understanding of our customers:
```

  * [Customer Feedback](https://community.mattermost.com/private-core/channels/customer-feedback)
  * [Customer Support](https://community.mattermost.com/private-core/channels/community)
* Begin watching training videos on [academy.mattermost.com ](https://academy.mattermost.com/)
* Begin reading [docs.mattermost.com](https://docs.mattermost.com/)

Team:

> * Meet the rest of team leads \(Developers, Designer, QA\) and join team meetings

Reviewer 2: Do you mean rest of the team the PM will work with most closely?

Author: Yes, updated to be "R&D team"

  * Ask questions to understand development, design, QA processes
* Meet with Amy \(Release Manager\) and read the release process doc
* Meet with Ian (CEO) \[[https://github.com/it33/readme/blob/master/README.md](https://github.com/it33/readme/blob/master/README.md/)\). This will be scheduled for you

> * Meet other Product Managers \(Jason Blais, Eric Sethna, Katie Wiersgalla, Aaron Rothschild, Ian Tao, Chen-I Lim\)

Reviewer 2: ```suggestion
* Meet other Product Managers \(Jason Blais, Eric Sethna, Katie Wiersgalla, Aaron Rothschild, Ian Tao, Chen-I Lim, Najla Dadmand\)
```

Reviewer 2: I would also add meeting with Product Marketing (Jack O'Brien today)

  * Learn about their areas of ownership
* Participate in PM, R&D, and Customer Obsession Meetings
* Jira
  * Sign up to Jira and post in the “Jira configuration” channel to be added to internal team
  * Get Jira training from PM buddy
    * [Mattermost Platform](https://mattermost.atlassian.net/projects/MM/issues/MM-10977?filter=allopenissues)
* Make your first edit and pull request on GitHub
  * Update documentation or product help text
* Small contributions
  * File a bug in Jira
  * Test a new feature on a test server and suggest an improvement
  * Leave feedback on a design proposal

Readings:
* The Advantage:
  * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
* The Five Dysfunctions of a Team: A Leadership Fable
  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)

> ## First 30 days

Reviewer 2: This is a lot. I'd like to see three outcomes for a PM 
1. Learn more about our users and customers, either via direct conversations with them and our team (which you have), or via quantitative data (e.g. via telemetry)
2. Ship their first feature. This can be a small enhancement, but would let them get exposed to the feature set, release process, and more.
3. Take ownership of KPIs and roadmap by end of day 30 -- depending on seniority and scope, this could be a subset of the roadmap they'll eventually own, e.g. roadmap for compliance, or for Desktop App. 

Let me know what you think?

Author: I personally think shipping first feature and taking ownership of KPIs and roadmap is too aggressive.  In my mind, they should be immersing themselves into learning and getting the flow and cadence of what we are doing and why.  I'd be hesitant to push them to get full ownership until they have the first 30 days under their belt.  

I do have full ownership in days 30-60, which takes them into building a feature from the start.

### Objective: Build foundational knowledge

### Training focus:

* Users and customers
* Product
  * Features/Functions
  * Technical
* Market and Competitive Landscape
* Team
  * People and Processes

### Activities:

> Learn about our customers and users

Reviewer: ```suggestion

Learn about our customers and users
```

* Support
  * Customers: Zendesk \(Sven, Support Lead)\)
    * Meet with Sven for Zendesk training and support overview
    * Join support standups for a week or two, and help out on support tickets
  * Community: GitHub, Forums \(Amy\)
* Customer Success \(Meet with Jenn\)
  * Overview of Customer Success team, who owns what
  * Current Customer Success processes
  * Typical challenges during customer onboarding
  * Join one CSM team meeting to meet the rest of the CS team
* Customer calls
  * Join calls with CSMs
  * Join calls with Sales team
* Feature requests
  * [Uservoice forums](https://mattermost.uservoice.com/)
  * [Process feedback in productboard](https://mattermost.productboard.com/insights/shared-inbox) 

> * Learn about the market

Reviewer 2: Any specific steps or links to learn about the market?

Author: Updating the formatting to make the steps here more obvious.

* Sales \(Regional VPs and Customer Engineers\)
  * Sales team overview
  * Sales team priorities
  * What competitors are we up against
  * How does Sales team position the product
  * What are typical challenges for the Sales team
  * Which sales do we win, which do we lose
  * What are the most common questions on configurations and trials
* Marketing \(Marketing Team Lead\)
  * Marketing team overview
  * What are the Marketing team’s priorities
  * What is our current messaging and positioning around Mattermost
  * What is needed from Product Managers
* Mini competitive research projects
    * Review of MS Teams, Slack, Discord, and other chat products

> * Spend a week doing customer support

Reviewer 2: I'm wondering if instead this would be a week doing community support in the forums? Given community support has lower expectations from the reporter, and there will be less process to learn (I think the support team uses tags and macros in certain support requests).

Author: Agreed.  I think that is a great adjustment. Updated.

  * Review documentation first and work with Support Lead or PM mentor to answer any questions documentation does not address

>   * Update docs with answers as appropriate

Reviewer: I'd like to see Documentation added to this list as a PM will work with a writer and I've found that an intro meeting really helps. It's also really beneficial to the writers to get fresh-eye feedback! :)

Author: Added to the first week as part of the R&D team introductions.

* Make your first edit and pull request on GitHub
  * Update documentation or product help text
* Small contributions
  * File a bug in Jira
  * Test a new feature on a test server and suggest an improvement
  * Leave feedback on a design proposal

Readings:
* The Advantage:
  * [https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view](https://drive.google.com/file/d/1zOV4nqVPxRHsQrxcqQwkGq2hCvB2CKJU/view)
* The Five Dysfunctions of a Team: A Leadership Fable
  * [https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing](https://drive.google.com/file/d/1yvLMW5tHGoTTjrq5VY17SCoeQoNXn9jA/view?usp=sharing)

## First 30 days

### Objective: Build foundational knowledge of our company, product, and market.

### Training focus:

* Users and customers
* Product
  * Features/Functions
  * Technical
* Market and Competitive Landscape
* Team
  * People and Processes

### Training outputs:
* Post detailed notes from customer calls  in [Salesforce](https://handbook.mattermost.com/operations/customer-success#logging-a-customer-feedback-in-salesforce)

> * Answer at least one question doing a week of community support in the Mattermost[forum](https://mattermost.uservoice.com/)

Reviewer 2: ```suggestion
* Answer at least one question during the week of community support in the Mattermost [forum](https://forum.mattermost.org/)
```
